config/captcha.captcha_point.user_register_form.yml

```diff
@@ -1,6 +1,6 @@
 uuid: 5111ef96-12c9-4b91-81de-b83fa65da65c
-langcode: nb
-status: true
+langcode: en
+status: false
```

Comment:
Vet ikke hvorfor den har blitt endret, men tipper vi vil beholde denne. Du kan sikkert bare endre den tilbake manuelt?

Reply:
Jeg er på mobilen akkurat nå og er ute hvor jeg ikke er nærheten av laptop. Jeg tror årsaken vat at den ble generert av `revert feature`-kommanddelen.

Reply:
Here is the log:
```
www-data@1c4259653be9:/app$ composer export
> ./vendor/bin/drush --root=$(pwd)/html cex -y
Differences of the active config to the export directory:

+-------------+-----------------------------------------------+-----------+
| Collection  | Config                                        | Operation |
+-------------+-----------------------------------------------+-----------+
|             | reroute_email.settings                        | Create    |
|             | core.extension                                | Update    |
|             | message.template.create_like_node_or_post     | Update    |
|             | block.block.socialblue_pagetitleblock_content | Update    |
|             | core.entity_form_display.post.post.profile    | Update    |
|             | core.entity_form_display.post.post.group      | Update    |
|             | core.entity_form_display.post.post.default    | Update    |
|             | core.entity_form_display.post.photo.profile   | Update    |
|             | core.entity_form_display.post.photo.group     | Update    |
|             | core.entity_form_display.post.photo.default   | Update    |
|             | views.view.activity_stream_group              | Update    |
|             | views.view.activity_stream_notifications      | Update    |
|             | views.view.activity_stream_profile            | Update    |
|             | captcha.captcha_point.user_register_form      | Update    |
| language.en | views.view.activity_stream_group              | Update    |
| language.en | views.view.activity_stream_notifications      | Update    |
| language.en | views.view.activity_stream_profile            | Update    |
| language.nb | message.template.create_like_node_or_post     | Update    |
| language.nb | views.view.activity_stream_group              | Update    |
| language.nb | views.view.activity_stream_notifications      | Update    |
| language.nb | views.view.activity_stream_profile            | Update    |
+-------------+-----------------------------------------------+-----------+

 // The .yml files in your export directory (../config) will be deleted and replaced with the active config.: yes.      

 [success] Configuration successfully exported to ../config.
www-data@1c4259653be9:/app$ 
```

Reply:
Vel, noen ganger skjer det endringer som man ikke vil ha. Bare fjern endringen, det er nok greiest. :)

Jeg fikk også den der når jeg oppgraderte til 1.19

Reply:
Så vil du at jeg skal kun reset denne filen `config/captcha.captcha_point.user_register_form.yml` (Altså ikke ta med kun den filen i denne PR)?